Sources/GRButton/GRButtonStyle.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  GRButtonStyle.swift
+//  GoodSwiftUI
+//
+//  Created by Lukas Kubaliak on 24/11/2024.
+//
+
+import SwiftUI
+
+public struct GRButtonStyle: ButtonStyle {
+    
+    // MARK: - Constants
+
+    private enum C {
+
+        static let minimalScale = CGFloat(0.95)
+        static let opacity = CGFloat(0.8)
+
+    }
+    
+    // MARK: - Size
+    
+    public enum Size {
```

Comment:
The size ENUM makes the whole style file a bit too long. I was thinking about extracting it into an extension in a separate file GRButtonStyleSize next to style.